docs/Installation.md

```diff
@@ -93,6 +93,28 @@ Make sure you avoid installing into:
 
 Create a Homebrew installation wherever you extract the tarball. Whichever `brew` command is called is where the packages will be installed. You can use this as you see fit, e.g. to have a system set of libs in the default prefix and tweaked formulae for development in `~/homebrew`.
 
+## Post-installation steps
+
+Before completing installation, Homebrew installer will provide some required "Next steps" instructions.
+These instructions configure your shell to evaluate the output of `brew shellenv`,
+which will load `brew` into your shell environment for use.
+
+While it's difficult to document the precise path for every shell,
+typically, what follows must be in your shell's `rc` or `profile` file:
+
+```sh
+eval "${HOMEBREW_PREFIX}/bin/brew shellenv)"
+```
+
+where `${HOMEBREW_PREFIX}` is the Homebrew installation directory.
+Replace this with the installation directory on your system.
```

Comment:
curious if there's a way to help people figure out what the installation directory is here. Maybe could just link to the FAQ entry? https://docs.brew.sh/FAQ#why-should-i-install-homebrew-in-the-default-location

something like "Replace this with the installation directory on your system ([listed here](https://docs.brew.sh/FAQ#why-should-i-install-homebrew-in-the-default-location))."

Reply:
How's this?

```suggestion
Replace this with the installation directory on your system.
See the [FAQ about default installation locations](FAQ.md#why-should-i-install-homebrew-in-the-default-location).
```